packages/react-ui-validations/stories/Switcher.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import { storiesOf } from '@storybook/react';
+import React from 'react';
+import { Button } from '@skbkontur/react-ui/components/Button';
+import { Switcher } from '@skbkontur/react-ui/components/Switcher/Switcher';
+
+import { ValidationContainer, ValidationInfo, ValidationWrapper, text } from '../src';
```

Comment:
Хелпер `text` не используется.
```suggestion
import { ValidationContainer, ValidationInfo, ValidationWrapper } from '../src';
```